src/js/components/Poll/PublicRegisterModal.vue

```diff
@@ -190,13 +190,13 @@ export default {
 
 			if (!this.isValidName) {
 				return {
-					result: t('polls', '{username} is invalid or reserved.', { username: this.userName }),
+					result: t('polls', 'Name {username} is invalid or reserved.', { username: this.userName }),
 					status: 'error',
 				}
 			}
 
 			return {
-				result: t('polls', '{username} is valid.', { username: this.userName }),
+				result: t('polls', 'Name {username} is valid.', { username: this.userName }),
```

Comment:
Dito. In doubt we could remove this and the 'Checking name ...' as well, return an empty string as default and skip the information. This reduces the noise.

Reply:
Where does this message appears? As a Pop up? Or below a form item?

Reply:
> In doubt we could remove this and the 'Checking name ...' as well, return an empty string as default and skip the information. This reduces the noise.

@dartcafe What about a wrong username?

> Where does this message appears?

@rakekniven See issue # 2430.

Reply:
> What about a wrong username? 

@Valdnet See also my last [comment](https://github.com/nextcloud/polls/pull/2443#discussion_r884582223).

Reply:
See my comment above.